src/components/locations-list/locations-list.tsx

```diff
@@ -0,0 +1,18 @@
+import LocationItem from '../location-item/location-item';
+
+type LocationItemProps = {
+  cities: string[];
+}
+
+function LocationsList({ cities }: LocationItemProps) {
+  const List = cities.map((item: string) => <LocationItem city={item} key={item} />);
```

Comment:
Ну это не класс, и не FC
```suggestion
  const list = cities.map((item: string) => <LocationItem city={item} key={item} />);
```